config.php

```diff
@@ -33,3 +33,4 @@
         ]
     ]
 ];
+?>
```

Comment:
this is not necessary but does not hurt if it is not followed by a newline like in this case. But if you accidentally add a newline after the `?>` tag PHP will generate output which is sent to the browser which might cause problems with sending headers. So I'd generally recommend not to add closing tags to files that contain only php code.

Reply:
Sadly I found that removing the `?>` closing tag causes issues within `/user/account/edit` for some weird reason which is why I readded the closing tag which seems to have fixed the issue.